bigquery/google/cloud/bigquery/table.py

```diff
@@ -684,6 +684,29 @@ def fetch_data(self, max_results=None, page_token=None, client=None):
         iterator._NEXT_TOKEN = 'pageToken'
         return iterator
 
+    def row_from_mapping(self, mapping):
+        """Convert a mapping to a row tuple using the schema.
+
+        :type mapping: dict
+        :param mapping: Mapping of row data.
+
+        :rtype: tuple
+        :returns: Tuple matching the table's schema
+        :raises: ValueError if table's schema is not set
```

Comment:
No:  you cannot perform `Table.create` without supplying a non-empty schema.  If it is empty, it means the user needs to call `Table.reload()` to fetch it, or else configure it manually.